e2e-tests/playwright/support/ui/components/channels/sidebar_left.ts

```diff
@@ -16,6 +16,34 @@ export default class ChannelsSidebarLeft {
     async toBeVisible() {
         await expect(this.container).toBeVisible();
     }
+
+    /**
+     * Clicks on the sidebar channel link with the given name.
+     * It can be any sidebar item name including channels, direct messages, or group messages, threads, etc.
+     * @param channelName
+     */
+    async goToItem(channelName: string) {
+        const channel = this.container.locator(`#sidebarItem_${channelName}`);
+        await channel.waitFor();
+        await channel.click();
+    }
+
+    /**
+     * Verifies 'Drafts' as a sidebar link exists in LHS.
+     */
+    async draftsExist() {
```

Comment:
Let's align this to the actual assertion being made since an element may exist but could be visible or not, and so I'd recommend: `draftsVisible` and the other as `draftsNotVisible`

Reply:
done